src/reports/common/useReportStyles.js

```diff
@@ -52,4 +52,7 @@ export default makeStyles((theme) => ({
       paddingBottom: 0,
     },
   },
+  actionCellIcons: {
```

Comment:
What does this do?

Reply:
It was supposed to display the two icons inline instead of below each other. But the better solution is to just add a new table row for this so this becomes unnnessesary. It's gone now